tests/testing_data/dts/collection_1.json

```diff
@@ -0,0 +1,41 @@
+{
+    "@context": {
+        "@vocab": "https://www.w3.org/ns/hydra/core#",
+        "dc": "http://purl.org/dc/terms/",
+        "dts": "https://w3id.org/dts/api#",
+        "tei": "http://www.tei-c.org/ns/1.0"
+    },
+    "@id": "general",
+    "@type": "Collection",
+    "totalItems": 3,
+    "title": "Collection Générale de l'École Nationale des Chartes",
+    "dts:dublincore": {
+        "dc:publisher": ["École Nationale des Chartes", "https://viaf.org/viaf/167874585"],
+        "dc:title": [
+            {"@language": "fre", "@value" : "Collection Générale de l'École Nationale des Chartes"}
```

Comment:
Weren't `@language` values in previous examples only 2 letters long, e.g., 'en' or 'la'?

Reply:
Not important technically :D